public/index.html

```diff
@@ -7,11 +7,13 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/icons/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="/icons/favicon-16x16.png">
     <link rel="manifest" href="/icons/site.webmanifest">
-    <link rel="mask-icon" href="/icons/safari-pinned-tab.svg" color="#333333">
+    <link rel="mask-icon" href="/icons/safari-pinned-tab.svg" color="#9b1b01">
```

Comment:
This color is not part of the official theme colors.

Reply:
done